src/components/entity/EntityResult.vue

```diff
@@ -1,55 +1,682 @@
+    this.selectedTopicFullCredentialSet.forEach((credSet) => {
+      //filter out all the raltionship credentials
+      fullCredentials.push(
+        ...credSet.credentials
+          .filter((cred) => !this.isRelationshipCred(cred))
+          .map((cred) => credOrRelationshipToDisplay(cred))
+      );
+    });
+    this.getRelationships.forEach((rel) => {
+      fullCredentials.push(credOrRelationshipToDisplay(rel));
+    });
+    return fullCredentials;
+  }
+
+  get entityCredentialsSorted(): Array<ICredentialDisplayType> | undefined {
+    const cpa = (
+      item1: ICredentialDisplayType,
+      item2: ICredentialDisplayType
+    ) => {
+      return (
+        (moment(item1.date_effective).isBefore(item2.date_effective) ? -1 : 1) *
+        this.credentialTimeOrder
+      );
+    };
+    return this.entityCredentials?.sort((cred1, cred2) => {
+      return cpa(cred1, cred2);
+    });
+  }
+
+  get entityState(): string | undefined {
+    if (this.selectedTopic === undefined) {
+      return undefined;
+    }
+
+    const state = selectFirstAttrItem(
+      { key: "type", value: "entity_status" },
+      this.selectedTopic?.attributes
+    );
+    return state?.type + "." + state?.value;
+  }
+
+  get entityRegistrationDate(): string | undefined {
+    return selectFirstAttrItem(
+      { key: "type", value: "registration_date" },
+      this.selectedTopic?.attributes
+    )?.value;
+  }
+
+  get entityEffectiveDate(): string | undefined {
+    return selectFirstAttrItem(
+      { key: "type", value: "entity_status_effective" },
+      this.selectedTopic?.attributes
+    )?.value;
+  }
+
+  get entityJurisdiction(): string | undefined {
+    const state = selectFirstAttrItem(
+      { key: "type", value: "home_jurisdiction" },
+      this.selectedTopic?.attributes
+    );
+    return "entity_type." + state?.value;
+  }
+
+  get entityRegistrationIssuer(): string | undefined {
+    return selectFirstAttrItem(
+      { key: "type", value: "entity_name" },
+      this.selectedTopic?.names
+    )?.issuer?.name;
+  }
+
+  get entityRegistrationIssuerUrl(): string | URL | undefined {
+    return selectFirstAttrItem(
+      { key: "type", value: "entity_name" },
+      this.selectedTopic?.names
+    )?.issuer?.url;
+  }
 
   async created(): Promise<void> {
     this.setLoading(true);
     const { sourceId } = this.$route.params;
+
     if (sourceId) {
       await this.fetchFormattedIdentifiedTopic({
         sourceId,
         type: "registration.registries.ca",
       });
       const topic: ITopic = this.$store.getters.selectedTopic;
       if (topic?.id) {
-        await this.fetchTopicCredentialSet(topic.id);
+        await Promise.all([
+          this.fetchRelationships(topic.id),
+          this.fetchTopicFullCredentialSet(topic.id),
+        ]);
       }
     }
     this.setLoading(false);
+    this.loadingCallBack();
   }
 }
 </script>
+<style scoped>
+.standing {
+  background-color: rgb(46, 133, 64);
```

Comment:
Add these coolers to the global variable style file and scope the styles using SASS